Add text/xml in Vulnerability_Management_-_Qualys PB for Quslys report (#16044)
demisto/content: Packs/qualys/ReleaseNotes/1_0_13.md

#### Playbooks
##### Vulnerability Management - Qualys (Job) - V2

> - Added text/XML file type to support Qualys report file In server 6.2.

Reviewer: ```suggestion
- Added the `text/XML` file type to support the Qualys report file for server version 6.2.
```
